eos-tech-support/eos-diagnostics

```diff
@@ -918,6 +984,13 @@ ARGV.forEach(function(arg) {
         filename = arg;
 });
 
+let image_version = trySpawn('attr -q -g eos-image-version /sysroot') +
+                    trySpawn('attr -q -g eos-image-version /');
+
+if (image_version.startsWith("payg")) {
+    paygFlag = true;
+}
+
```

Comment:
Some payg image filenames do not start with "payg", but have "payg" as part of the personality string.

That said, we don't need to try to identify a payg image, we can try to fetch this information on all systems.